<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBinEntryDialogBase</class>
 <widget class="QDialog" name="EditBinEntryDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bin Color</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="TextLabel">
       <property name="text">
        <string>Bin type:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="TextLabelBinType">
       <property name="toolTip">
        <string>Type of binning edited</string>
       </property>
       <property name="text">
        <string>Bin type:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="TextLabel_2">
       <property name="text">
        <string>Bin list:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditBinList">
       <property name="toolTip">
        <string>Enter Bin or Bin list to use this color. E.g: 5-10,13,17</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="TextLabel_2_3">
       <property name="text">
        <string>E.g:  3-30</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="TextLabel_2_2">
       <property name="text">
        <string>Color:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="GColorPushButton" name="pushButtonEditColor">
       <property name="toolTip">
        <string>Click to edit color!</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="PushButtonOk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>59</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PushButtonCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>59</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>GColorPushButton</class>
   <extends>QPushButton</extends>
   <header>gcolorpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditBinList</tabstop>
  <tabstop>pushButtonEditColor</tabstop>
  <tabstop>PushButtonOk</tabstop>
  <tabstop>PushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
